addon/components/x-range-input.js

```diff
@@ -61,14 +59,13 @@ export default Ember.Component.extend(PropertyBindings, {
   value: 0,
 
   /**
-   * On any `input` event, copy the numeric value of the DOM element
-   * onto the `value` property of the component so that it can be
-   * bound to and from.
+   * On any `input` event, take the component and the element `value` and send
+   * it in an action.
    *
    * @private
    */
-  input: function() {
-    this.set('value', Number(this.get('element.value')).valueOf());
+  input() {
+    this.sendAction('action', this, this.get('element.value'));
```

Comment:
is `element.value` a string? Also, if `element.value` is the empty string, it will return `0`